sockeye/training.py

```diff
@@ -203,6 +204,10 @@ def fit(self,
 
             if self.state.updates > 0 and self.state.batches % (
                     self.config.checkpoint_interval * self.config.update_interval) == 0:
+                # ############
```

Comment:
weird formatting. Do we need it? :)

Reply:
we have this similarly in Sockeye 1: 
https://github.com/awslabs/sockeye/blob/master/sockeye/training.py#L698

Anyway, we can remove it. Just wanted to point out the location where checkpointing is happening.